src/libs/ContractHelper.sol

```diff
@@ -35,14 +35,38 @@ library ContractHelper {
                             nonce_ == 0x00
                                 ? abi.encodePacked(bytes1(0xd6), bytes1(0x94), account_, bytes1(0x80))
                                 : nonce_ <= 0x7f
-                                ? abi.encodePacked(bytes1(0xd6), bytes1(0x94), account_, uint8(nonce_))
-                                : nonce_ <= 0xff
-                                ? abi.encodePacked(bytes1(0xd7), bytes1(0x94), account_, bytes1(0x81), uint8(nonce_))
-                                : nonce_ <= 0xffff
-                                ? abi.encodePacked(bytes1(0xd8), bytes1(0x94), account_, bytes1(0x82), uint16(nonce_))
-                                : nonce_ <= 0xffffff
-                                ? abi.encodePacked(bytes1(0xd9), bytes1(0x94), account_, bytes1(0x83), uint24(nonce_))
-                                : abi.encodePacked(bytes1(0xda), bytes1(0x94), account_, bytes1(0x84), uint32(nonce_))
+                                    ? abi.encodePacked(bytes1(0xd6), bytes1(0x94), account_, uint8(nonce_))
+                                    : nonce_ <= 0xff
```

Comment:
what is the reason for this change? can it be in a separate Pr with a short info

Reply:
prettier auto formatting